src/components/WwwFrame/WwwPage.vue

```diff
@@ -68,6 +69,7 @@ export default {
 		preFetch(_, client) {
 			return Promise.all([
 				client.query({ query: hasEverLoggedInQuery }),
+				client.query({ query: myKivaForAllUsersQuery }),
```

Comment:
While I like the idea of caching this once, I think we should be cautious with adding this to ALL page requests. We have many uses that will never need this flag (search bots, blog pages, campaigns, etc.). Try to identify a way to share some caching but not run this query on all pages. If we need to instead load the settings directly on a few pages, we should be fine losing some code simplification for better performance.